### PR TITLE
Add animate on scroll

### PR DESCRIPTION
**Pull request by ogdenstudios**
@baudoinalkali - this is the animation you mentioned from the Paradox website. I'm using the [Animate on scroll](https://michalsnik.github.io/aos/) library, which is pretty popular and ubiquitous for that kind of thing. 

The cool thing about the library is it has a lot of built in transitions out of the box. I chose the one that looked closest to what you showed, but you may want to tweak it a little bit. I'll show you where in an inline comment. 

The other thing to be aware of - the AOS animations are usually pretty lightweight and unobtrusvie. But in some cases they will mess with existing styles. It's good you didn't want to apply it to the phones, but basically I would do a thorough check through the site to see if things are behaving unexpectedly, and consider turning the AOS data attribute on or off. 

You'll see a lot of additions that look like this: `data-aos="zoom-in-up"`. 

Basically, anything with that attribute does the `zoom-in-up` animation from AOS. You can find-and-replace it if you want to use a different animation. Or you can remove it, or add it to places that you want to change. In addition to removing, adding, or changing the animation, there are some underlying configuration options you can mess with (again, I'll show you where that is).

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
 import '../styles/globals.css'
 import 'tailwindcss/tailwind.css'
+import { useEffect } from "react";
+import AOS from "aos";
+import "aos/dist/aos.css";
 
 function MyApp({ Component, pageProps }) {
+  // Since we're integrating the animate on scroll library with React,
```
Comment (ogdenstudios) [inline pages/_app.js:8]: @baudoinalkali - here is where you could set different global configuration options for AOS. [You can refer to the usage guide on their GitHub](https://github.com/michalsnik/aos#-how-to-use-it) for the different available options.

File: pages/index.js [hunk 2]
```diff
@@ -78,7 +78,7 @@ export default function Home() {
 
       </div>
 
-      <div className="grid grid-cols-2 text-center lg:grid-cols-6 max-w-7xl m-auto align-center justify-between items-center gap-y-14 pt-24 gap-x-10 lg:gap-x-20 px-8">
+      <div data-aos="zoom-in-up" className="grid grid-cols-2 text-center lg:grid-cols-6 max-w-7xl m-auto align-center justify-between items-center gap-y-14 pt-24 gap-x-10 lg:gap-x-20 px-8">
```
Comment (ogdenstudios) [inline pages/index.js:81]: @baudoinalkali - basically you'll see this wherever I added the AOS animation. Again, you can change which animation you want using [any of the pre-existing ones they put together](https://michalsnik.github.io/aos/), or you can remove them, or move this to different elements. It's easy to change it up, and it's pretty robust. I got you started, so hopefully you should be able to make any further adjustments as necessary once I'm wrapped up on the project entirely.

**Comment by ogdenstudios** [comment]
@baudoinalkali - I just tweaked this so the individual home page blobs for applicants/employers use AOS. I think that basically resolves exactly what they're looking for on that one.

**Comment by baudoinalkali** [comment]
@ogdenstudios I like it, let me run it by the client, I'm thinkin perhaps it should come into view a tad more and the animation should be a bit slower, but I'll get their opinion. One thing I'm noticing is the animations seem to not always show. It's weird and I'm trying to mess around to get it replicating consistently. For instance if I navigate through some pages not all the animations on the homepage function. If I open a new tab and navigate to the link it works as intended every time.

**Comment by ogdenstudios** [comment]
@baudoinalkali, you can tweak those in the initializer. 

To get it to come into view more, you can change the `offset` field. And the speed of the animation is set to `duration`. 

As for not always running, you may need to either change the `startEvent` to work with some Next.js event, or perhaps there's a better way to structure the `useEffect` hook. 

[This GitHub issue](https://github.com/michalsnik/aos/issues/574) has more details there as well. 

Let me know if you get stuck there, but hopefully that gives you all the info you need to tweak it with their feedback.

**Comment by ogdenstudios** [comment]
@baudoinalkali - pretty sure I fixed the intermittent bug. Here's my best understanding of what's going on (I'm still not 100% sure if this is what was going on. It's just my best guess, and seems to be working). 

1. Initially I had the AOS initialize in `pages/_app.js`, which I assumed would run on every page load. It might, it might not. I'm not sure. I just ended up extracting it to its own hook called `useAos`, in the utilities folder at `utilities/aos.js`. Then I had every page explicitly call it. 
2. That change alone didn't fix things, but then I added an event listener to the `window` object for the `load` event. I pulled it from [this comment](https://github.com/michalsnik/aos/issues/110#issuecomment-313558842) in the GitHub issue you sent me. I felt good about it because the maintainer said [that there might be issues if an app is dynamically resizing something after AOS initializes](https://github.com/michalsnik/aos/issues/110#issuecomment-413954172). That seems most likely to me, since `useEffect` doesn't always run on document ready, rather it runs on first render. 
3. So now, we initialize AOS on render, and at the same time, we register an event listener for the window's `load` event to refresh AOS once the document is ready - that will ensure that AOS is both initialized, and correctly calculates the information it needs to work. 

The config for this is now in `utilities/aos.js`, but it works basically the same. 

Check it out and let me know what you think.
